patches/chrome-installer-linux-debian-build.sh.patch

```diff
@@ -20,15 +20,15 @@ index 306b59c138b7cb5df21ad6e9d502bb7a46a104c6..f66764e6637e5fdc882ebf4046a702f1
    -O > "${STAGEDIR}/DEBIAN/control"
    rm -f "${DEB_CONTROL}"
  }
-@@ -49,7 +49,7 @@ stage_install_debian() {
-   # Always use a different name for /usr/bin symlink depending on channel.
-   # First, to avoid file collisions. Second, to make it possible to
-   # use update-alternatives for /usr/bin/google-chrome.
--  local USR_BIN_SYMLINK_NAME="${PACKAGE}-${CHANNEL}"
-+  local USR_BIN_SYMLINK_NAME="${PACKAGEANDCHANNEL}"
+@@ -41,7 +41,7 @@ prep_staging_debian() {
+   install -m 755 -d "${STAGEDIR}/DEBIAN" \
+     "${STAGEDIR}/etc/cron.daily" \
+     "${STAGEDIR}/usr/share/menu" \
+-    "${STAGEDIR}/usr/share/doc/${USR_BIN_SYMLINK_NAME}"
++    "${STAGEDIR}/usr/share/doc/${PACKAGEANDCHANNEL}"
```

Comment:
Since we no longer patch the value of `${USR_BIN_SYMLINK_NAME}`, we need to make sure that the doc directory uses the package name (`${PACKAGEANDCHANNEL} == 'brave-browser'`) and not the binary name (`${USR_BIN_SYMLINK_NAME} == 'brave-browser-stable'`).